test/extended/builds/pipeline.go

```diff
@@ -200,7 +201,7 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 			br.AssertSuccess()
 
 			g.By("confirm all the log annotations are there")
-			_, err = jenkins.ProcessLogURLAnnotations(oc, br)
+			err = jenkins.ProcessLogURLAnnotations(oc, br, !useSnapshotImage)
```

Comment:
this means the PR tester won't ever confirm we haven't broken the log annotations....   can't we make this instead discriminate on the jenkins image version being tested?